maltd-frontend/src/components/base/GeneralButton/GeneralButton.css

```diff
@@ -1,8 +1,58 @@
 .general-button {
-  width: 150px;
-  height: 40px;
+  width: 150px !important;
+  height: 40px !important;
+  float: right !important;
+  color: #ffffff !important;
+  font-family: "Open Sans", sans-serif !important;
 }
 
-.align-right {
-  float: right;
+/* Bootstrap .btn CSS override */
+.btn:focus {
+  box-shadow: none !important;
+}
+
+/* Bootstrap .btn-primary CSS override */
+.btn-primary {
+  background-color: #4788c4 !important;
+  border-color: #979797 !important;
+  opacity: 0.75 !important;
+}
+
+.btn-primary:active,
+.btn-primary:hover,
+.btn-primary:focus {
+  background-color: #4788c4 !important;
+  border-color: #979797 !important;
+  opacity: 1 !important;
+}
+
+.btn-primary:disabled,
```

Comment:
would like to discuss if all these custom styles are required or if we can live with using more of the styling that comes out of the box with the library we are using. It sorta depends on how strict the client is about styling matching exactly with the mockups (which I don't think they are). I would ideally prefer minimizing this complexity of styling - let's discuss

Reply:
I'm okay with either options. Since we have limited base components and those are really what requires much CSS override, once we have them ready, the composite ones should be fine. Anyways, let's discuss! 👍